content/en/docs/gke/deploy/delete-cli.md

```diff
@@ -63,12 +67,12 @@ If you need fine-grained control, refer to
 [Config Connector: Keeping resources after deletion](https://cloud.google.com/config-connector/docs/how-to/managing-deleting-resources#keeping_resources_after_deletion)
 for more details.
 
-After deleting cnrm resources for a managed project, you can revoke IAM perssion
+After deleting cnrm resources for a managed project, you can revoke IAM permission
```

Comment:
> After deleting cnrm resources for a managed project, you can revoke IAM permission

Do you know if it's common to refer to it as "CNRM" or "cnrm"? So far, I've seen some mentions, such as https://github.com/GoogleCloudPlatform/cnrm-blueprints but I can't find the definition. Perhaps, we should spell it out first (e.g. "Cloud Native Resource Management (CNRM). WDYT?

Reply:
same thing on line 92 ("cnrm")

Reply:
Good point!
I'll leave it to a centralized issue: https://github.com/kubeflow/gcp-blueprints/issues/175, because we are using the word fairly inconsistently now. I think it's worth centralizing this effort